src/main/scala/NumberFormat.scala

```diff
@@ -0,0 +1,39 @@
+package CSVValidation
+
+import Errors.NumberFormatError
+import com.ibm.icu.text.DecimalFormat
+
+case class NumberFormat(pattern:Option[String], groupChar: Option[Char] = None, decimalChar: Option[Char] = None) {
+  private var df:DecimalFormat = new DecimalFormat()
+  private val decimalFormatSymbols = df.getDecimalFormatSymbols
+  decimalFormatSymbols.setInfinity("INF")
```

Comment:
What was this needed for?

Reply:
In default configuration DecimalFormat does not recognise `INF` symbols